timeline/tests/test_models.py

```diff
@@ -269,6 +269,29 @@ def test_repr_no_project_no_user(self):
         expected = "ProjectEvent('N/A', 'test_event', 'N/A')"
         self.assertEqual(repr(self.event), expected)
 
+    def test_find_name(self):
```

Comment:
You wrote earlier you had some problems with timeline events and testing. Is that still an issue and if so, can you describe it to me in detail? I can help out.

Reply:
I've already fixed it! Thanks